client/app/bundles/HelloWorld/components/ResturantsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+
+export default class ResturantsList extends React.Component {
+    componentDidMount(){
+        console.log("mounted");
+        console.log(this.props);
+    };
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            resturants: JSON.stringify(this.props.resturants)
```

Comment:
Yes, the stringify wasn't necessary eventually since I changed something silly I did back in the controller. Won't be here in my next commit